### DISCUSSION TITLE
Integrating with Home Assistant

### DISCUSSION THREAD

**Discussion by jsmith79** [Geektoolkit/Dynaframe3#119] [General]
Hey Guys, Just wondering how many people are integrating their Dynaframe with home assistant and how are they doing it.
There are any number of ways it can be done , im just curious if anyone has done it and how their experience has been with it.

I would love to have automations to turn on and off based on motion and time and also change playlists.

**Comment by Geektoolkit** [comment]
I'm doing quite a bit of that using Node red currently and the http API I made. Mine turn on/off with my zwave switch.

I'd like to know if there's a better way to do it...possibly exposing an MQTT section or some such. 

Open to suggestions!

**Reply by astlgit** [reply]
@Geektoolkit would love it if you could do a video or some sort of instruction on how you do this. It would be super helpful.

**Comment by dankinchen** [comment]
I just purchased 6 Raspberry Pi 4's (w/ SunFounder 10.1 screens), and I intend to run videos on each of these. I plan to use Amazon Alexa to run each of them. So I am very curious myself. I am currently watching all of @Geektoolkit videos hoping to implement this myself this week.
